kevinene91/Fast-Food-Fast-API
# [Feature #160782584]and auth endpoints

### What does this PR do ? 
Setup Auth endpoints
### Description of Task 
- Endpoinst 
`POST api/v2/auth/login`
`POST api/v2/auth/sigunp`

### How to manually test this 
lCone the repo
`git clone  https://github.com/kevinene91/Fast-Food-Fast-API.git`
Navigate to the folder 
`cd Fast-Food-Fast-Api`
switch to  this branch 
`git checkout  ft-user-auth-db-1607825842`
Create virtualenv 
`virtualenv venv`
Activate the enviroment 
`source venv/bin/activate`
install requirements 
`pip install -r requirements.txt`
create databse with postgres
export enviroment variables for database
`export SECRET_KEY="your secret key"
export DB_URL="dbname=fastfoodfast user=user password= host=localhost"
export TEST_DB_URL="dbname=yourtestb user=user password= host=localhost"`

navigate to tests/v2 folder
run `python pytest test_auth_endpoint.py` should pass tests
on the root folder run `python run.py`
Test using Postman 

### Pivotal Tracker 
[160782584](https://www.pivotaltracker.com/n/projects/2193832/stories/160782584)

### Screenshot 
![login](https://user-images.githubusercontent.com/11676467/46153051-0d9da000-c27b-11e8-848d-9ea1c9f00eae.png)
![db_signup](https://user-images.githubusercontent.com/11676467/46153085-227a3380-c27b-11e8-8c81-939232351cc7.png)
